README.md

```diff
@@ -73,47 +75,40 @@ macOS tools:
 
 Unix tools:
 
-* [Git] for version control
 * [htop] for viewing process information in the CLI
 * [OpenSSL] for Transport Layer Security (TLS)
 * [The Silver Searcher] for finding things in files
-* [vim] as a handy in-terminal editor for rainy days
+* [tmux] for saving project state and switching between projects
 * [Zsh] as your shell
 * [Oh My Zsh] framework for Zsh
 
-[Git]: https://git-scm.com/
```

Comment:
Installing git in this script is unnecessary since we do the manual install earlier in the README.